INSTRUCTION.md

```diff
@@ -0,0 +1,14 @@
+kubectl get pv -n todoapp
+kubectl get pvc -n todoapp
+kubectl get pods -o wide -n todoapp
+kubectl exec -it <pod-name> -n todoapp -- sh
+   ls 
+   cd 
+kubectl exec -it <pod-name> -n todoapp --sh
```

Comment:
There is a typo in the command. It should be '-- sh' instead of '--sh'. Ensure there is a space between '--' and 'sh'.